v2/datastream-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToBigQuery.java

```diff
@@ -30,9 +30,10 @@
 import com.google.cloud.teleport.v2.transforms.UDFTextTransformer.InputUDFOptions;
 import com.google.cloud.teleport.v2.transforms.UDFTextTransformer.InputUDFToTableRow;
 import com.google.cloud.teleport.v2.values.FailsafeElement;
-import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Set;
+import java.util.regex.Pattern;
+import com.google.common.base.Splitter;
```

Comment:
(Sorry, missed this!) Imports need to be ordered alphabetically. Otherwise, it will cause a checkstyle violation. So the `Splitter` import should come between `import com.google.cloud.teleport.v2.values.FailsafeElement;` and `import java.util.HashSet;`.

For future reference, you can run checkstyle with `mvn checkstyle:checkstyle` from the module with the change or from v2/ with `mvn -pl m0,m1,...,mN checkstyle:checkstyle`, so for this PR, you could run 

`mvn -pl common,datastream-to-bigquery checkstyle:checkstyle`

Reply:
Sure!